packages/material-ui/src/Slider/Slider.js

```diff
@@ -618,8 +624,7 @@ const Slider = React.forwardRef(function Slider(props, ref) {
   });
 
   const handleTouchStart = useEventCallback((event) => {
-    if (event.cancelable && iOS) {
-      // Workaround as Safari has partial support for touchAction: 'none'.
+    if (!touchActionSupport) {
```

Comment:
Why not just check `event.cancelable`? That way we don't accidentally leave this untouch if we change when we add passive listeners or not.

Reply:
> We get a warning in Chrome if we don't do so.

document this please

Reply:
And we should investigate it further. This doesn't seem right.

Reply:
> And we should investigate it further. This doesn't seem right.

It doesn't seem right, but I don't think that we should care either. Will it make a difference?